.github/workflows/ci.yml

```diff
@@ -0,0 +1,36 @@
+# This is a basic workflow to help you get started with Actions
+
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the "main" branch
```

Comment:
I don't call my main branch main. Do we need to use the name that is aliased? From what perspective are we looking for the alias?